addons/calendar/static/tests/calendar_notification_tests.js

```diff
@@ -0,0 +1,205 @@
+          id: 'prout',
+          channel: ['db_name', 'calendar.alarm', 7],
+          message: [
+            {
+                "alarm_id": 1,
+                "event_id": 2,
+                "title": "Meeting",
+                "message": "Very old meeting message",
+                "timer": 20 * 60,
+                "notify_at": '1978-04-14 12:45:00',
+            }
+          ],
+        }]);
+      }
+      if (route === '/calendar/notify') {
+        return Promise.resolve(true);
+      }
+    };
+
+    const fakeActionService = {
+      name: "action",
+      deploy() {
+        return {
+          doAction(actionId) {
+            assert.step(actionId);
+            return Promise.resolve(true);
+          },
+          loadState(state, options) {
+            return Promise.resolve(true);
+          },
+        };
+      },
+    };
+    testConfig.serviceRegistry.remove("action");
+    testConfig.serviceRegistry.add("action", fakeActionService);
+
+    const webClient = await createWebClient({
+      testConfig,
+      legacyParams: { serviceRegistry: legacyServicesRegistry },
+      mockRPC,
+    });
+
+    await nextTick();
+
+    assert.containsOnce(webClient.el, '.o_notification_body');
+    assert.strictEqual(
+      webClient.el.querySelector('.o_notification_body .o_notification_content').textContent,
+      "Very old meeting message"
+    );
+
+    await click(webClient.el.querySelectorAll('.o_notification_buttons .btn')[1]);
+    assert.verifySteps(["calendar.action_calendar_event_notify"]);
+    assert.containsNone(webClient.el, ".o_notification");
+
+    webClient.destroy();
+  });
+
+  QUnit.test('can listen on bus and displays notifications in DOM and click Snooze', async (assert) => {
+    assert.expect(3);
+
+    const mockRPC = (route, args) => {
+      if (route === '/longpolling/poll') {
+        return Promise.resolve([{
+          id: 'prout',
+          channel: ['db_name', 'calendar.alarm', 7],
+          message: [
+            {
+                "alarm_id": 1,
+                "event_id": 2,
+                "title": "Meeting",
+                "message": "Very old meeting message",
+                "timer": 20 * 60,
+                "notify_at": '1978-04-14 12:45:00',
+            }
+          ],
+        }]);
+      }
+      if (route === '/calendar/notify') {
+        return Promise.resolve(true);
+      }
+    };
+
+    const webClient = await createWebClient({
+      testConfig,
+      legacyParams: { serviceRegistry: legacyServicesRegistry },
+      mockRPC,
+    });
+
+    await nextTick();
+
+    assert.containsOnce(webClient.el, '.o_notification_body');
+    assert.strictEqual(
+      webClient.el.querySelector('.o_notification_body .o_notification_content').textContent,
+      "Very old meeting message"
+    );
+
+    await click(webClient.el.querySelectorAll('.o_notification_buttons .btn')[2]);
+    assert.containsNone(webClient.el, ".o_notification");
+
+    webClient.destroy();
```

Comment:
snooze is different that ack, you should assert we don't call the ack route (like you did in the first test)